src/common/state/ColorControlPointList.code

```diff
@@ -77,23 +77,37 @@ Definition:
 float
 ColorControlPointList::EvalCubicSpline(float t, const float *allX, const float *allY, int n) const
 {
-    if((allX[0] > t) || (allX[n-1] < t))
-        return 0.f;
+    // If before first sample, return first sample value.
```

Comment:
Shouldn't this be used to update the C file (xml2atts) ?

Reply:
Oh, thats right. I need to go do that.

Reply:
Ok, thats done now.